ProjectSettings/TagManager.asset

```diff
@@ -10,6 +10,8 @@ TagManager:
   - PlayerProjectile
   - EnemyProjectile
   - Trap
+  - Half-Heart
```

Comment:
If we decide to use specialized tags like this for the hearts, and we decide to add more hearts in the future (e.g., the golden heart you proposed), we'll have to keep checking the tags separately (e.g., `if player hit this heart or if player hit this heart or if player hit that other heart etc`). Instead, I propose we create a single heart prefab tagged as a Heart, and give that prefab a script with a `public` value denoting how much health it gives. That way, when the player simply collides with a `Heart` (we don't care what type on the player's end), we can simply do this in Player.cs: `ChangeHealthBy(theHeart.health)`.